js/jquery.mapael.js

```diff
@@ -55,6 +64,9 @@
                 , zoomCenterX = 0
                 , zoomCenterY = 0
                 , previousPinchDist = 0;
+                
+            // add plugin class name on element
+            $self.addClass(pluginName);
```

Comment:
This allows the plugin to be deterministic:
- declaring externally that mapael is assigned to this element
- allow specific CSS customization aimed at mapael (without fearing name
collision)
On Nov 19, 2015 8:53 PM, "Vincent Brouté" <notifications@github.com> wrote:

> In js/jquery.mapael.js
> <https://github.com/neveldo/jQuery-Mapael/pull/124#discussion_r45391579>:
>
> > @@ -55,6 +64,9 @@
> >                  , zoomCenterX = 0
> >                  , zoomCenterY = 0
> >                  , previousPinchDist = 0;
> > +
> > +            // add plugin class name on element
> > +            $self.addClass(pluginName);
>
> What is the purpose of adding the plugin name as a class name on the
> element ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neveldo/jQuery-Mapael/pull/124/files#r45391579>.
>